### ISSUE TITLE
Hosting Environment Check blank during setup.php with PHP error

### ISSUE THREAD

**Issue by A1Gaming** [postfixadmin/postfixadmin#633]
Hello,

During the Postfix Admin setup process, where you instruct in the "install.txt" to open the setup.php file, I have an issue with the Hosting Environment Check process. It is blank, with zero checks. I cannot setup the Admin account or finish the installation process.
![Screenshot_2022-06-30_10-40-12](https://user-images.githubusercontent.com/36427041/176709123-6224ed02-c544-439d-83c8-2da42df01ca8.png)

I did notice a fatal PHP error message, in the www-error.log file PHP-FPM logged, on line 1629 in functions.inc.php.
![Screenshot_2022-06-30_10-37-55](https://user-images.githubusercontent.com/36427041/176709230-65342368-524e-4e40-875b-5de695d9dea3.png)
![Screenshot_2022-06-30_10-20-10](https://user-images.githubusercontent.com/36427041/176709381-237599f1-a9fb-4d86-b735-56e0e171ff4b.png)

My setup is a dedicated server, on a LAN, running OS Rocky Linux 8.6, Postfix 3.5.8, Dovecot 2.3.19.1, MariaDB 10.6.8, PHP 8.1.7.
My root directory for the Postfix Admin files is in /srv/postfix-admin/

I am sorry if I am lacking info, or giving too much at one time. I appreciate your time and efforts in this project.

**Comment by DavidGoodwin** [comment]
Hi - thanks for the screenshots and checking the error log!

You need to (somehow) install the PHP PDO driver. Hunt around for a package named something like php-pdo or php-mysql - once installed, restart Apache, and it may behave?

**Comment by drdelaney** [comment]
for Rocky Linux 8, it will most likely need to be `sudo dnf install php-pdo`.  You will need to also restart both apache and php-fpm (default on RL8). `sudo systemctl restart php-fpm httpd`

**Comment by A1Gaming** [comment]
Thanks for the replies!

To DavidGoodwin:
I went ahead and installed php-pdo (I remember doing this 2 years ago when I first setup Postfix-Admin, but I forgot about it!) and the Hosting Environment checks did proceed as normal! However, I needed to install php-mysqld to correct the error "cannot connect to database, no driver found" and php-imap to correct the error "cannot auto-create folders...".

Everything works now! It updated my existing database from 1840 to 1844, apparently without issue. It found my original super admin user too. 
![Screenshot_2022-06-30_12-00-48](https://user-images.githubusercontent.com/36427041/176725568-b67c2c05-d2ff-44ca-9c18-4a8f24fad5f4.png)

To drdelaney:
Thanks for the input! I appreciate the help!

Thanks for helping everyone!
